src/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -101,7 +101,6 @@ static_library("SiWx917") {
 
     public_deps += [
       "${chip_root}/src/crypto",
-      "${mbedtls_root}:mbedtls",
```

Comment:
I see below that SiWx917 still uses a lot of mbedTLS includes, without this dependencies won't that break something?

Reply:
Everything that is needed is pulled in the conditional "source set" base on the crypto_flavor (tinycrypt or psa)